src/linear_samplers/block_col_rand_cyclic.jl

```diff
@@ -9,6 +9,8 @@ After all blocks are called, a new random ordering is created.
 - `n_blocks::Int64`, Variable that contains the number of blocks overall.
 - `order::Vector{Int64}`, The order that the blocks will be used to generate updates.
 - `blocks::Vector{Vector{Int64}}`, The vector containing all the groupings of column indices.
+- `block_size::Int64`, Variable that represents the smallest sketching block size in iterations. 
```

Comment:
Should be `Union{Int64, Nothing}`; please update preceding docstring arguments to have the full type allowed by the method.